website/docs/d/database_connection.html.markdown

```diff
@@ -0,0 +1,324 @@
+---
+layout: "ibm"
+page_title: "IBM : ibm_database_connection"
+description: |-
+  Get information about database_connection
+subcategory: "The IBM Cloud Databases API"
```

Comment:
Can we use the same subcategory of existing databse resource /datasource  (Cloud databases) doc so that it gets grouped under same when the docs are onboarded to registry
![Uploading Screenshot 2022-03-09 at 12.30.30 PM.png…]()

Reply:
Fixed